File: terre07.exs
# Taille d'une chaine

System.argv()
|> case do
  [arg] ->
    arg
    |> String.to_charlist()
    |> Enum.reduce(0, fn _char, count ->
      count + 1
    end)
    |> IO.puts()

  _ ->
    IO.puts("Usage: elixir terre07.exs \"<sentence>\"")
    System.stop(1)
end

File: terre07.js
// Taille d'une chaine
const args = process.argv.slice(2)
if (args != 1) {
    console.log("Usage: node terre06.js \"<sentence>\"")
    process.exit(1)
}

const sentence = args[0]

let length = 0
while (sentence[length]) {
    length++
}

console.log(length)
